src/core_plugins/console/public/src/app.js

```diff
@@ -2,114 +2,113 @@ var $ = require('jquery');
 
```

Comment:
Looks like my editor automagically formatted this file to our eslint guidelines, which is why it seems so different, but it's actually not. If you diff these two files in a tool that allows you to ignore whitespace, you'll see that there were actually only a handful of changes, not as many as it seems below.